mergify_cli/ci/cli.py

```diff
@@ -187,3 +190,21 @@ async def junit_process(  # noqa: PLR0913
         tests_target_branch=tests_target_branch,
         files=files,
     )
+
+
+@ci.command(
+    help="""Give the list scope impacted by changed files""",
+    short_help="""Give the list scope impacted by changed files""",
+)
+@click.option(
+    "--config",
+    "config_path",
+    required=True,
+    type=click.Path(exists=True),
+    default=pathlib.Path("mergify-ci.yml"),
```

Comment:
I think most people expected this to be a dotfile, are we happy with that? Just wondering.

Reply:
We should keep it consistant with `.mergify.yml`